Assets/Scripts/Config/ConfigLoader.cs

```diff
@@ -12,6 +13,22 @@ public static class ConfigLoader {
   // Relative path to the default simulator configuration.
   private const string SimulatorConfigRelativePath = "simulator.pbtxt";
 
+  // Map from the interceptor type to the static configuration file.
+  private static readonly Dictionary<Configs.InterceptorType, string> InterceptorStaticConfigMap =
+      new() {
+        { Configs.InterceptorType.Hydra70, "hydra70.pbtxt" },
+        { Configs.InterceptorType.Micromissile, "micromissiles.pbtxt" },
+      };
+
+  // Map from the threat type to the static configuration file.
+  private static readonly Dictionary<Configs.ThreatType, string> ThreatStaticConfigMap = new() {
+    { Configs.ThreatType.Quadcopter, "quadcopter.pbtxt" },
+    { Configs.ThreatType.Ucav, "ucav.pbtxt" },
+    { Configs.ThreatType.Brahmos, "brahmos.pbtxt" },
+    { Configs.ThreatType.Ascm, "ascm.pbtxt" },
+    { Configs.ThreatType.Fateh110B, "fateh_110b.pbtxt" },
+  };
+
```

Comment:
Hmm this is showing compile-time binding of static configurations right? I think this is counter to our philosophy of users being able to add their own models when they are working with just the binary.

Like Kris will not be able to make and add his own model files anymore after this change?

Reply:
Let me address this in the simulation config PR - the static config PR does not implement compile-time binding of static configs.

Reply:
Got it, that makes sense to me 👍